src/app/log/[id]/Log.view.tsx

```diff
@@ -46,47 +48,39 @@ const LogView = ({
       />
       <S.LogContainer>
         <LogMapSection
-          masilsData={masilsData}
+          masilData={masilData}
           currentPinIndex={currentPinIndex}
           mapCenter={mapCenter}
-          handlePinIndex={handlePinIndex}
+          handlePinIndex={handleCurrentPinIndex}
           handleClickCenter={handleClickCenter}
         />
         <S.LogContentLayout>
           <Tab
-            style={{
-              fontSize: `${FONT_SIZE.H6}`,
-              fontWeight: `${FONT_WEIGHT.BOLD}`,
-            }}
+            className="logTab"
             tabContents={TAB_CONTENTS}
-            tabClickHandler={(index) => {
-              setTabIndex(index);
-            }}
+            tabClickHandler={handleClickTab}
             focusedTab={tabIndex}
           />
 
           <S.LogContentSection className="scrollbar-hide">
             {tabIndex === TabType.Memo && (
               <LogMemo
-                distance={1100}
-                totalTime={3600}
+                masilData={masilData}
                 userInfo={userInfo}
-                address={`${masilsData.depth1} ${masilsData.depth2}`}
-                masilDate={masilsData.startedAt}
               />
             )}
             {tabIndex === TabType.Pin && (
               <LogPin
-                pins={masilsData.pins}
+                pins={masilData.pins}
                 currentPinIndex={currentPinIndex}
-                handlePinIndex={handlePinIndex}
+                handlePinIndex={handleCurrentPinIndex}
               />
             )}
           </S.LogContentSection>
 
-          <Link href="/post/:id">
+          <Link href={`/post/create?logId=${logId}`}>
             <Button
-              width="calc(100% - 3rem)"
+              width="calc(100% - 4rem)"
               textColor={Theme.lightTheme.white}
               buttonColor={Theme.lightTheme.green_500}
               style={{

```

Comment:
[P2]
저번에 들어보니 저희 theme을 아래와 같이 사용해야한다고 하더라구요.

함수 컴포넌트 상단에서 theme hoook을 선언해주고
아래에서 받아 사용하는 방식이라고 합니다!

```suggestion
const theme = useTheme()


              width="calc(100% - 4rem)"
              textColor={theme?.white}
              buttonColor={theme?.green_500}
              style={{

```

Reply:
그렇군요 감사합니다 !! 😇
그럼 서버 컴포넌트에서는 `theme style`을 사용하려면 어떻게 해야 할까요??! 🤔

Reply:
tailwind에 변수를 선언하고 꺼내야할것같네요 ㅠ 🤦